src/components/Common/StyledLink.js

```diff
@@ -96,23 +95,14 @@ const StyledLink = ({ external, to, href, children, ...props }) => {
         rel: 'noopener noreferrer'
       }
     : {}
-  if (to) {
-    return (
-      <Anchor {...externalProps} to={to} {...props}>
-        {children}
-      </Anchor>
-    )
-  }
+
+  const redirectProps = to ? { to } : { as: 'a', href }
 
   return (
-    <Anchor {...externalProps} as="a" href={href} {...props}>
+    <Anchor {...externalProps} {...props} {...redirectProps}>
```

Comment:
I'm umming and erring about this change. It's perfectly fine at the moment and isn't broken so the first thing that comes to mind is "if it aint broke don't fix it". Generally ternaries are considered to take a bit longer to understand than the humble `if () {}` conditional. 

It's up to you with this but personally I'm very happy with what we have at the moment as its clear and works.

Reply:
`ternaries are considered to take a bit longer to understand than the humble if () {} conditional.`
Mhh. I didn't know that. Shame because I love ternaries ❤️ Let's discuss at stand-up. We may revert them

Reply:
Just did some reading on the topic and some other devs say that ternary is faster. Ultimately it does not seem to be everyone's opinion but what also comes out is that the speed difference is negligible. I like Ternary readability. Once again, we can discuss at SU